apps/test/unit/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_infoTest.js

```diff
@@ -104,4 +105,61 @@ describe('Workshop Enrollment School Info', () => {
         .filterWhere(col => col.text().includes('Scholarship Teacher?'))
     ).to.have.length(0);
   });
+
+  it('shows questions for CSP returning teachers', () => {
+    let workshopEnrollmentSchoolInfo = shallow(
+      <WorkshopEnrollmentSchoolInfo
+        enrollments={[]}
+        accountRequiredForAttendance={true}
+        scholarshipWorkshop={false}
+        onDelete={() => {}}
+        onClickSelect={() => {}}
+        workshopCourse="CS Principles"
+        workshopSubject={SubjectNames.SUBJECT_CSP_FOR_RETURNING_TEACHERS}
+        numSessions={5}
+        permissionList={new Permission(['ProgramManager'])}
+      />,
+      {context}
+    );
+
+    ['Years Teaching CS', 'Taught AP Before?', 'Planning to teach AP?'].forEach(
```

Comment:
Note that I could not for the life of me figure out how to get a version of this to work with exact string matching, so I removed the "Years Teaching" question as it matched two headers instead of one.